UITests/Tests/AxeScanAllTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using System;
+using System.Linq;
+using System.Text.Json;
+using System.IO;
+using System.Collections.ObjectModel;
+
+namespace UITests.Tests
+{
+    [TestClass]
+    public class AxeScanAll : TestBase
+    {       
+        public static readonly string jsonUri = "ms-appx:///../../../../../../WinUIGallery/DataModel/ControlInfoData.json";
```

Comment:
Should we instead add the file to the project to copy it to the output directory?